lib/xcribe/config.ex

```diff
@@ -1,121 +1,35 @@
 defmodule Xcribe.Config do
   @moduledoc false
 
-  alias Xcribe.{MissingInformationSource, UnknownFormat}
-
   @valid_formats [:api_blueprint, :swagger]
 
-  @doc """
-  Return true if serve mode is enabled.
-
-  If no config was given the default is false.
-
-  To configure server mode:
-
-      config :xcribe, [
-        serve: true
-      ]
-  """
-  def serving?, do: get_xcribe_config(:serve, false) == true
-
-  @doc """
-  Return the file name to output generated documentation.
-
-  If no config was given the default names are `api_doc.apib` for Blueprint
-  format and `openapi.json` for Swagger format.
-
-  To configure output name:
-
-      config :xcribe, [
-        output: "custom_name.json"
-      ]
-  """
-  def output_file, do: get_xcribe_config(:output, default_output_file())
-
-  @doc """
-  Return the format for documentation.
-
-  Default is `:api_blueprint`. 
-
-  To configure the documentation format:
+  def active?, do: System.get_env("XCRIBE_ENV")
```

Comment:
```suggestion
  def active?, do: System.get_env("XCRIBE_ENV") in ["1", "true", "TRUE"]
```
Wouldn't be better to return a boolean here, so in other places we don't need to use some form of shortcut?

Reply:
Agree, done 👍🏽